corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -533,12 +533,15 @@ def _report_filters(self):
 
         def _make_report_filter(conf):
             col_id = self.data_source_properties[conf["property"]]['column_id']
-            return {
+            ret = {
                 "field": col_id,
                 "slug": col_id,
                 "display": conf["display_text"],
                 "type": filter_type_map[conf['format']]
             }
+            if conf['format'] == 'Date':
+                ret.update({'compare_as_string': True})
```

Comment:
Within the report builder I'm not really passing around the types, but I certainly could for the case meta properties and form questions. Do you think I should use these known types for the indicators and filters when possible? It will add a little complexity to the code, but shouldn't take very long to write.